### DISCUSSION TITLE
Status and future of tox-uv?

### DISCUSSION THREAD

**Discussion by paravoid** [tox-dev/tox-uv#83] [General]
Thank you for tox-uv, @gaborbernat. uv is refreshing to use, and the tox/uv combination is especially so. Being able to run tox against multiple Python versions is so easy now :smile: 

As you may remember, I am the Debian maintainer for tox. I am wondering whether I should package tox-uv for Debian.

Would you describe it as stable or experimental? What are your long-term intentions for it? Are you planning to merge it back into the tox codebase? If that's in the short- to mid-term plans, I could avoid the separate package and all that.

Thanks again!

**Comment by gaborbernat** [comment]
It is stable. No plans to merge it back.
